chirp/drivers/ft450d.py

```diff
@@ -1001,162 +816,198 @@ def get_settings(self):
+        rx = RadioSettingValueInteger(0, 10, val)
+        tmp = "Resume scan (secs)"
+        rs = RadioSetting("scn_res", tmp, rx)
         tab.append(rs)
 
-        rs = RadioSetting("split", "TX/RX Frequency Split",
-                          RadioSettingValueBoolean(_settings.split))
+        rx = RadioSettingValueBoolean(_settings.clk_sft)
+        tmp = "CPU clock shift"
+        rs = RadioSetting("clk_sft", tmp, rx)
         tab.append(rs)
 
-        rs = RadioSetting("qspl_f", "Quick-Split freq offset (KHz)",
-                          RadioSettingValueInteger(-20, 20, _settings.qspl_f))
+        rx = RadioSettingValueBoolean(_settings.split)
+        tmp = "TX/RX Frequency Split"
+        rs = RadioSetting("split", tmp, rx)
         tab.append(rs)
 
-        rs = RadioSetting("emergen", "Alaska Emergency Mem 5167.5KHz",
-                          RadioSettingValueBoolean(_settings.emergen))
+        val = _settings.qspl_f
+        rx = RadioSettingValueInteger(-20, 20, val)
+        tmp = "Quick-Split freq offset (KHz)"
+        rs = RadioSetting("qspl_f", tmp, rx)
         tab.append(rs)
 
-        rs = RadioSetting("stby_beep", "PTT release 'Standby' beep",
-                          RadioSettingValueBoolean(_settings.stby_beep))
+        rx = RadioSettingValueBoolean(_settings.emergen)
+        tmp = "Alaska Emergency Mem 5167.5KHz"
+        rs = RadioSetting("emergen", tmp, rx)
         tab.append(rs)
 
-        options = ["ATAS", "EXT ATU", "INT ATU", "INTRATU", "F-TRANS"]
-        rs = RadioSetting("tuner", "Antenna Tuner",
-                          RadioSettingValueList(options,
-                          options[_settings.tuner]))
+        rx = RadioSettingValueBoolean(_settings.stby_beep)
+        tmp = "PTT release 'Standby' beep"
+        rs = RadioSetting("stby_beep", tmp, rx)
         tab.append(rs)
 
-        rs = RadioSetting("rfpower", "RF power (watts)",
-                          RadioSettingValueInteger(5, 100, _settings.rfpower))
+        if self.MODEL != "FT-450":
+            options = ["ATAS", "EXT ATU", "INT ATU", "INTRATU", "F-TRANS"]
+            optn = options[_settings.tuner]
+            rx = RadioSettingValueList(options, optn)
+            tmp = "Antenna Tuner"
+            rs = RadioSetting("tuner", tmp, rx)
+            tab.append(rs)
+
+        val = _settings.rfpower
+        rx = RadioSettingValueInteger(5, 100, val)
+        tmp = "RF power (watts)"
+        rs = RadioSetting("rfpower", tmp, rx)
         tab.append(rs)      # End of _do_general_settings
 
 
     def _do_cw_settings(self, cw):        # - - - CW - - -
         _settings = self._memobj.settings
         _beacon = self._memobj.beacontext
 
-        rs = RadioSetting("cw_dly", "CW break-in delay (ms * 10)",
-                          RadioSettingValueInteger(0, 300, _settings.cw_dly))
+        val = _settings.cw_dly
+        rx = RadioSettingValueInteger(0, 300, val)
+        tmp = "CW break-in delay (ms * 10)"
```

Comment:
I understand that you are refactoring existing code but I find the "ms * 10" part confusing. Wouldn't it better if the user choose a value in milliseconds and the program convert it? If I wanted a delay of 100 ms should I choose 10 or 1000? Should I read the manual for the radio or trust that chirp does the right thing?